organization.go

```diff
@@ -213,6 +215,9 @@ type OrganizationCreateOptions struct {
 	// Optional: SendPassingStatusesForUntriggeredSpeculativePlans toggles behavior of untriggered speculative plans to send status updates to version control systems like GitHub.
 	SendPassingStatusesForUntriggeredSpeculativePlans *bool `jsonapi:"attr,send-passing-statuses-for-untriggered-speculative-plans,omitempty"`
 
+	// Optional: If enabled, SendPassingStatusesForUntriggeredSpeculativePlans needs to be false.
+	AggregatedCommitStatusEnabled *bool `jsonapi:"attr,aggregated-commit-status-enabled"`
```

Comment:
Oh, these Update/Create options should continue to be omitempty, because in this case the empty pointer omits the attribute!